src/lib.rs

```diff
@@ -102,6 +102,8 @@ use typenum::{Cmp, Greater, Less, U0, U1, U2, U3, U4, U5, U6, U7, U8, U9, U10,
 
 mod num_traits_impl;
 
+pub use num_traits_impl::*;
```

Comment:
It may be clearer to replace this with `pub use num_traits_impl::ParseFixedError` as that's the only public item in that module.

Reply:
I found that with just `use` the trait implementation were not accessible from a third-party crate. Changing to `pub use` fixed things for me. Weird, but I couldn't find a better solution.